_shared/project/requirements/prod.in

```diff
@@ -8,6 +8,10 @@ sqlalchemy
 psycopg2
 alembic
 {% endif %}
+{% if cookiecutter.get("_directory") == "pyramid-app" and cookiecutter.get("postgres") == "yes" %}
+pyramid-tm
+zope.sqlalchemy
+{% endif %}
```

Comment:
Add the `pyramid-tm` and `zope.sqlalchemy` requirements that're needed to integrate the SQLAlchemy session and transaction into Pyramid (only for Pyramid apps with `postgres=yes`)